docs/tutorials/basic/symbol.md

```diff
@@ -328,4 +498,20 @@ ex.forward()
 ex.outputs[0].asnumpy()
 ```
 
+## How Efficient Is the Symbolic API?
+
+In short, it is designed to be very efficient in both memory and runtime.
+The major reason for introducing the Symbolic API is to bring the efficient C++
```

Comment:
This section is confusing a number of concepts. this sentence inparticular doesn't make much sense.